### PR TITLE
make all mentions of NEO4J_HOME and /bin the same

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1500.surge.sh

Updated pages:
- [Recover admin user and password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/authentication-authorization/password-and-user-recovery)
- [Backup modes](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/modes)
- [Restore a database backup](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Managing databases in a cluster](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/databases)
- [Intra-cluster encryption](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/setup/encryption)
- [Default file locations](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/file-locations)
- [Configure plugins](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/plugins)
- [Set an initial password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/set-initial-password)
- [Sharding data with the <code>copy</code> command](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/database-administration/composite-databases/sharding-with-copy)

File: modules/ROOT/pages/performance/memory-configuration.adoc [hunk 1]
```diff
@@ -102,7 +102,7 @@ Estimate the total size of the database files.
 
 [source, shell]
 ----
-$neo4j-home> bin/neo4j-admin server memory-recommendation
+$NEO4J_HOME> bin/neo4j-admin server memory-recommendation
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/performance/memory-configuration.adoc:105]: just to double check: Do we need `$NEO4J_HOME>` here?

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 2]
```diff
@@ -432,7 +432,7 @@ For example:
 
 [source, shell, role=noplay]
 ----
-neo4j_home$ bin/neo4j-admin database import full --nodes import/node_header.csv,'import/node_data_\d{1,5}.csv' databasename
+neo4j_homebin/neo4j-admin database import full --nodes import/node_header.csv,'import/node_data_\d{1,5}.csv' databasename
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:435]:
```suggestion
bin/neo4j-admin database import full --nodes import/node_header.csv,'import/node_data_\d{1,5}.csv' databasename
```

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 3]
```diff
@@ -1339,7 +1339,7 @@ actors-header.csv  actors.csv.zip  movies-header.csv  movies.csv.gz  roles-heade
 
 [source, sh]
 ----
-neo4j_home$ bin/neo4j-admin database import --nodes import/movies-header.csv,import/movies.csv.gz --nodes import/actors-header.csv,import/actors.csv.zip --relationships import/roles-header.csv,import/roles.csv.gz
+neo4j_homebin/neo4j-admin database import --nodes import/movies-header.csv,import/movies.csv.gz --nodes import/actors-header.csv,import/actors.csv.zip --relationships import/roles-header.csv,import/roles.csv.gz
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:1342]:
```suggestion
bin/neo4j-admin database import --nodes import/movies-header.csv,import/movies.csv.gz --nodes import/actors-header.csv,import/actors.csv.zip --relationships import/roles-header.csv,import/roles.csv.gz
```

File: modules/ROOT/pages/performance/memory-configuration.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/performance/memory-configuration.adoc:105]: No, of course not. Good catches! Replace without checking afterwards doesn't seem to work 😄

File: modules/ROOT/pages/performance/space-reuse.adoc [hunk 4]
```diff
@@ -169,7 +169,7 @@ Use the `system` database and stop the `neo4j` database before running the comma
 +
 [source, shell]
 ----
-$neo4j-home/bin$> ./neo4j-admin database copy neo4j neo4jcopy1 --compact-node-store --verbose
+$NEO4J_HOME/bin$> ./neo4j-admin database copy neo4j neo4jcopy1 --compact-node-store --verbose
```
Comment (renetapopova) [inline modules/ROOT/pages/performance/space-reuse.adoc:172]:
```suggestion
bin/neo4j-admin database copy neo4j neo4jcopy1 --compact-node-store --verbose
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1500.surge.sh

Updated pages:
- [Recover admin user and password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/authentication-authorization/password-and-user-recovery)
- [Backup modes](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/modes)
- [Restore a database backup](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Managing databases in a cluster](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/databases)
- [Intra-cluster encryption](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/setup/encryption)
- [Default file locations](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/file-locations)
- [Configure plugins](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/plugins)
- [Set an initial password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/set-initial-password)
- [Sharding data with the <code>copy</code> command](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/database-administration/composite-databases/sharding-with-copy)

File: modules/ROOT/pages/performance/space-reuse.adoc [hunk 5]
```diff
@@ -61,7 +61,7 @@ Let's use the Cypher Shell command-line tool to add 100k nodes and then see how
 +
 [source, shell]
 ----
-$neo4j-home/bin$> ./cypher-shell -u neo4j -p <password>
+$NEO4J_HOME/bin$> ./cypher-shell -u neo4j -p <password>
```
Comment (renetapopova) [inline modules/ROOT/pages/performance/space-reuse.adoc:64]:
```suggestion
bin/cypher-shell -u neo4j -p <password>
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1500.surge.sh

Updated pages:
- [Recover admin user and password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/authentication-authorization/password-and-user-recovery)
- [Backup modes](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/modes)
- [Restore a database backup](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Managing databases in a cluster](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/databases)
- [Intra-cluster encryption](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/setup/encryption)
- [Default file locations](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/file-locations)
- [Configure plugins](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/plugins)
- [Set an initial password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/set-initial-password)
- [Sharding data with the <code>copy</code> command](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/database-administration/composite-databases/sharding-with-copy)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1500.surge.sh

Updated pages:
- [Recover admin user and password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/authentication-authorization/password-and-user-recovery)
- [Backup modes](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/modes)
- [Restore a database backup](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Managing databases in a cluster](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/databases)
- [Intra-cluster encryption](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/clustering/setup/encryption)
- [Default file locations](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/file-locations)
- [Configure plugins](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/plugins)
- [Set an initial password](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/configuration/set-initial-password)
- [Sharding data with the <code>copy</code> command](https://neo4j-docs-operations-1500.surge.sh/operations-manual/5/database-administration/composite-databases/sharding-with-copy)

**Review by NataliaIvakina — APPROVED** [review]
I think, it looks good. Thank you, @renetapopova!

**Pull request merged**